Packs/TwitterSOARx/TestPlaybooks/playbook-Remove_Excluded_Accounts.yml

```diff
@@ -0,0 +1,141 @@
+id: Remove Excluded Accounts
+version: -1
+name: Remove Excluded Accounts
+description: Removes accounts from twitter search results if the accounts are in a
+  specified list in XSOAR.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 289c0f25-3233-4ff2-88a5-7cb1870ccdf0
+    type: start
+    task:
+      id: 289c0f25-3233-4ff2-88a5-7cb1870ccdf0
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "2":
+    id: "2"
+    taskid: e2be8345-2e4a-43d1-8b73-a5d6a1171279
+    type: regular
+    task:
+      id: e2be8345-2e4a-43d1-8b73-a5d6a1171279
+      version: -1
+      name: Get Users
+      description: Search for Users
+      script: TwitterSOARx|||twitter-get-users
+      type: regular
+      iscommand: true
+      brand: TwitterSOARx
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      name:
+        simple: '"Palo Alto Networks"'
+      page:
+        simple: "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: 4845df85-350e-4f26-8c8d-403c329f8e9b
+    type: title
+    task:
+      id: 4845df85-350e-4f26-8c8d-403c329f8e9b
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "5":
+    id: "5"
+    taskid: 025432d6-af62-49d4-8282-e4a5b3cccfce
+    type: regular
+    task:
+      id: 025432d6-af62-49d4-8282-e4a5b3cccfce
+      version: -1
+      name: Remove Excluded Users
+      scriptName: TwitterRemoveExcludedAccounts

```

Comment:
what is this script?

Reply:
This script is a sample playbook that returns results of a Twitter search (from the twitter-get-users command) and filters out accounts that are in a given List in XSOAR. This allows a user to search for users and filter accounts that are known to be harmless to them so they do not have to see accounts that they have added to a list for any given reason (such as having investigated the account, determining it is harmless and wanting to not see it again)